lib/goodcity/rake_logger.rb

```diff
@@ -0,0 +1,16 @@
+module Goodcity
+  class RakeLogger
+    def initialize(task)
+      @task = task
+      @logger = Logger.new("#{Rails.root}/log/rake_log.log")
+    end
+
+    def log_info(msg)
+      @logger.info("task=#{@task} #{msg}")
+    end
```

Comment:
Keep parity with the Logger class - use the same method names and overload them to inject the rake task name.

    def info(msg)
      super("task=#{@task} #{msg}")
    end

    def debug(msg)
      super("task=#{@task} #{msg}")
    end